src/database/DataTypes/Period.js

```diff
@@ -33,12 +33,21 @@ export class Period extends Realm.Object {
 
   requisitionsForOrderType(program, orderType) {
     return this.requisitions.filtered(
-      'program.id = $0 && orderType = $1',
+      'program.id = $0 && orderType = $1 && type == "request"',
       program.id,
       orderType.name
     ).length;
   }
 
+  customerRequisitionsForOrderTypeAndName(program, orderType, name) {
```

Comment:
2 nitpicks: 

- Might be clearer if you change `requisitionsForOrderType` to `supplierRequisitionsForOrderType`
- The method names I was expecting the actual realm results object to returned, not the length. But I can see that only length is used so 🤷‍♂️

Reply:
Great point. I made them more descriptive, but now a bit long. 🤷 


1791f74

Reply:
Yea the length I was a little worried about which is why I had approved regardless haha